examples/cadvisor-influx/juttle_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "adapters": {
+        "juttle-influxdb-adapter": {
```

Comment:
This should just be 'influxdb' now. I've already changed the others in a PR I'll soon merge.